bedrock/firefox/templates/firefox/new/email/scene2-base.html

```diff
@@ -0,0 +1,114 @@
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/base.html" %}
+
+{% block canonical_urls %}
+  {# the "scene 2" page should set canonical to the /firefox/new/ page to give that
+     page the SEO weight of links directly to "scene 2".
+     see https://github.com/mozilla/bedrock/pull/5463 #}
+  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/">
+{% endblock %}
+
+{% block page_og_url %}{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/{% endblock %}
+
+{% block page_title %}{{ _('Download Firefox') }} — {{ _('Free Web Browser') }}{% endblock %}
+
+{# All stylesheets are loaded in extrahead to serve legacy IE basic styles #}
+{% block extrahead %}
+  {{ super() }}
+  {{ css_bundle('firefox_new_scene2_email') }}
+{% endblock %}
+
+{% block body_id %}firefox-new-scene2{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% if LANG == 'en-CA' %}
+  {% set firefox_logo_title = 'Firefox (CA)' %}
+  {% set mozilla_logo_title = 'Mozilla (CA)' %}
+{% elif LANG == 'en-GB' %}
+  {% set firefox_logo_title = 'Firefox (UK)' %}
+  {% set mozilla_logo_title = 'Mozilla (UK)' %}
+{% else %}
+  {% set firefox_logo_title = 'Firefox' %}
+  {% set mozilla_logo_title = 'Mozilla' %}
+{% endif %}
+
+{% block content %}
+<div class="status">
+  <div class="content">
+    {# fallback_url is replaced by the platform download link via JS, but if
+      something fails the user should still get a link to a working download path. #}
+
+    {% set id, fallback_url = 'direct-download-link', url('firefox.all') %}
+    <p>
+    {% trans %}
+      Your download should begin automatically. Didn’t work? <a id="{{ id }}" href="{{ fallback_url }}">Try downloading again</a>.
+    {% endtrans %}
+    </p>
+  </div>
+</div>
+
+<main class="main">
+  <div class="content">
+    <div class="header-logos">
+      <h2><a class="firefox" href="{{ url('firefox') }}" title="{{ firefox_logo_title }}">{{ high_res_img('logos/firefox/logo-quantum-wordmark-white.png', {'alt': firefox_logo_title, 'width': '133', 'height': '50'}) }}</a></h2>
+      <h2><a class="mozilla" href="{{ url('mozorg.home') }}" title="{{ mozilla_logo_title }}"><img src="{{ static('img/logos/mozilla/wordmark-dark.svg')}}" alt="{{ mozilla_logo_title }}" width="101" height="34"></a></h2>
+    </div>
+
+    <div class="content-main">
+      <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+        {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
+      </div>
+
+      <h1 class="headline">{% block headline %}{% endblock %}</h1>
+      <p class="tagline">{{ _('Get informative tips, tricks and product announcements delivered to your inbox.') }}</p>
+
+      <div class="c-newsletter">
+        {{ email_newsletter_form(
```

Comment:
Not something we necessarily need to fix right now, but what do people see if they don't get the newsletter translated?

Reply:
Good question... since this is English only for now it's not an issue but I don't know what the longer term plan is. There's talk of soon redesigning the default /new as well as /thanks so things are likely to change. When we have a new default thanks page for all locales we'll definitely need a non-newsletter fallback. I'd suggest Fx mobile, like on the current page.